js/components/agency_component_preview.jsx

```diff
@@ -8,15 +8,30 @@ import PrettyUrl from './pretty_url';
 import { AgencyComponent } from '../models';
 
 
-function AgencyComponentPreview({ agencyComponent }) {
+function AgencyComponentPreview({ onAgencySelect, agencyComponent, isCentralized }) {
   const description = AgencyComponent.agencyMission(agencyComponent);
   const requestUrl = `/request/agency-component/${agencyComponent.id}/`;
+  const onSelect = () => onAgencySelect(agencyComponent.agency);
 
   return (
     <div className="agency-preview usa-grid-full">
       <div className="usa-width-one-whole">
-        <h2>{agencyComponent.agency.name}</h2>
-        <h3>{agencyComponent.title}</h3>
+        {
+          !isCentralized && (
+            <h2>
+              <span // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
Why a span instead of an `a` tag?
Alternatively, we could save an element and just give the `h2` these attributes.

Reply:
h2 is a block element, so it has 100% width. So all that empty space to the left of the content would also be clickable. The span makes it so that only the text itself is clickable. Similarly, the caret would appear on the far right, not at the end of the content if we just moved everything to the h2.

Reply:
sorry, reading too fast... an `a` tag doesn't really make sense to me because it's not a link. It's more of a button than a link, hence the `role="button"`.